models/runner/README.md

```diff
@@ -46,6 +46,12 @@ Compress-Archive -Path .\models\runner_win\, .\models\gabon\, .\models\serengeti
 
 ## Running in dev mode directly
 
+These instructions were tested on Debian 9.13 with Python 3.6.9.
+You need to install the following system dependencies:
+```sh
+sudo apt install libmkl_rt libtorch-dev
```

Comment:
should be `sudo apt install libmkl-rt libtorch-dev`

Reply:
I also needed to install this package: `libpython3.6-dev` but `sudo apt-get install libpython3.6-dev` on ubuntu 20.04 didn't work, I had to do: `sudo add-apt-repository ppa:deadsnakes/ppa \
&& sudo apt update \
&& sudo apt install python3.6`
and then I could install that package